lib/parsers/javascript.js

```diff
@@ -70,8 +70,13 @@ function _addComment(visited, data, commentValue, commentLoc, path, nodeLoc, inc
       });
 
       if (path.parentPath && path.parentPath.node) {
-        context.code = data.source.substring
-          .apply(data.source, path.parentPath.node.range);
+        var parentNode = path.parentPath.node;
+        if (parentNode.range) {
```

Comment:
We updated our babel deps pretty recently. I think we should be able to remove this.